### PR TITLE
add osx_arm64 support

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ test:
   commands:
     - test -f "$PREFIX/lib/liboclgrind-rt-icd${SHLIB_EXT}"         # [unix]
     - if not exist %LIBRARY_BIN%\\oclgrind-rt-icd.dll exit 1       # [win]
-    - oclgrind --help
+    - oclgrind --help  # [build_platform == target_platform]
```
Comment (isuruf) [inline recipe/meta.yaml]: why? tests are disabled anyways for osx-arm64

Comment (matthiasdiener) [reply recipe/meta.yaml]: I'm not sure I understand - where are the osx_arm64 tests disabled?

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by matthiasdiener** [comment]
Thanks, LGTM!

**Pull request merged**